SixTest/elensidealthck4d/README

```diff
@@ -1,3 +1,9 @@
+Tests:
+- thin4d and thin6d use the same single and structure elements. Test based on LHC lattice with HEL inserted at IP4.
+- thck4d adds an element 'thicky' with a nonzero length to force the thick 4D tracking to the thin4d/thin6d test
+- thck6d is based on thick6dblocks as the LHC example could not be used as for 6D thick tracking linear elements can not be grouped in blocks -> LHC example as defined fails.
+
+Pythong script:
```

Comment:
with spelling you mean replace HEL by ELEN?

On 2016-07-15 02:19, Kyrre Ness Sjøbæk wrote:
>
> In SixTest/elensidealthck4d/README 
> <https://github.com/SixTrack/SixTrack/pull/81#discussion_r70931027>:
>
> > @@ -1,3 +1,9 @@
> > +Tests:
> > +- thin4d and thin6d use the same single and structure elements. Test based on LHC lattice with HEL inserted at IP4.
> > +- thck4d adds an element 'thicky' with a nonzero length to force the thick 4D tracking to the thin4d/thin6d test
> > +- thck6d is based on thick6dblocks as the LHC example could not be used as for 6D thick tracking linear elements can not be grouped in blocks -> LHC example as defined fails.
> > +
> > +Pythong script:
>
> Splelling :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/SixTrack/SixTrack/pull/81/files/30837dd7de1d16430c535256618efe8a8e9f6309..0fb4f3869ad5770f65849c21d6f6030b4d2dff89#r70931027>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AIKzcWYnxG-9PXNV-fQHRKlPPaSkbn65ks5qVzRzgaJpZM4I6PV7>.
>